linera-service/src/cli_wrappers/local_net.rs

```diff
@@ -588,21 +590,23 @@ impl LocalNet {
         bail!("Failed to start {nickname}");
     }
 
-    async fn run_server(&mut self, validator: usize, shard: usize) -> Result<Child> {
+    async fn initialize_storage(&mut self, validator: usize, shard: usize) -> Result<String> {
         let shard_str = match self.database {
             Database::RocksDb => format!("_{}", shard),
```

Comment:
That really won't work with RocksDB anymore then, as soon as one uses blobs?

Reply:
Shouldn't we remove the option from the code then. (Can be a separate PR, of course, but it would be good to agree on the plan and have a ticket for it.)

Reply:
Ah yes! I was planning on doing that. Issue for it: #2068